fmn/rules/taskotron.py

```diff
@@ -5,7 +5,8 @@
 RELEASE_CRITICAL_TASKS = [
```

Comment:
Suddenly I wonder if it would make sense to have these in a configuration file :)